### PR TITLE
Avoid simple passwords (e.g. email as password)

### PR DESCRIPTION
**Pull request by pscheit**
PR changes that you can't use your email address or your username as a password anymore.

We are aware that it's hard in general to get people to use really good passwords, but at least this crosses out the really dumb mistakes.

### CODE REVIEW

File: tests/Controller/ProfileControllerTest.php [hunk 1]
```diff
@@ -21,6 +31,13 @@ public function setUp(): void
         parent::setUp();
     }
 
+    public function tearDown(): void
+    {
+        static::getContainer()->get(Connection::class)->rollBack();
+
+        parent::tearDown();
+    }
+
```
Comment (pscheit) [inline tests/Controller/ProfileControllerTest.php:40]: this is pretty easy to forget and would make a good case for a nice sideffect on the next functional test.

I would suggest we base class some stuff in the functional tests into a ControllerTestCase and let all Controller tests extend from this?  
Would allow to introduce a getEM() on tests as well

Would of course do this in another PR.

Comment (Seldaek) [reply tests/Controller/ProfileControllerTest.php:40]: Yeah sounds reasonable

**Comment by Seldaek** [comment]
Thanks
